apps/test/unit/code-studio/pd/workshop_enrollment/enroll_formTest.js

```diff
@@ -55,14 +57,49 @@ describe('Enroll Form', () => {
           workshop_course="CS Principles"
           first_name={props.first_name}
           email={props.email}
+          previous_courses={props.previous_courses}
           onSubmissionComplete={props.onSubmissionComplete}
+          collect_demographics={false}
         />
       );
     });
 
     it('does not display role question', () => {
       expect(enrollForm.find('#role')).to.have.length(0);
     });
+
+    it('does not display previous courses question', () => {
+      expect(enrollForm.find('#previous_courses')).to.have.length(0);
+    });
+
+    it('does not display replace existing question', () => {
+      expect(enrollForm.find('#replace_existing')).to.have.length(0);
+    });
+  });
+
+  describe('CSP Enroll Form', () => {
```

Comment:
Can we change the name to indicate how this case is different from the one above? "CSP EnrollForm with demographics" maybe?